### PR TITLE
Updated Lithuanian layout pack, merged it into master

### PR DESCRIPTION
**Pull request by rimas-kudelis**
This PR contains the following changes:
- Adds the AOSP Lithuanian dictionary
- Changes the keyHeight for the default Lithuanian layout from small to normal (I found small quite uncomfortable for fast typing)
- Drops the separate Lithuanian branch by merging it into master
- Updates all graphic assets

I have released this update in Google Play already (https://play.google.com/store/apps/details?id=org.herrlado.ask.languagepack.lithuanian) and marked it as beta for now.

Regarding dictionaries: I've left both the AOSP and the one we used originally in the source, but with AOSP, our originally produced dictionary might actually be unnecessary (especially since it's isn't really Apache licensed).

The release process currently expects a key named "lithuanian" in the key store. I used Lado's key to sign it (key fingerprint is A0:C8:20:B7:F4:42:8B:0A:E4:C1:B0:15:F5:FF:2F:E9:36:A9:29:0F), which I suspect is probably same as the key used for Czech language pack. @menny I can forward the key to you if you don't have it.

### CODE REVIEW

**Comment by rimas-kudelis** [comment]
@menny is there anything that's preventing you from reviewing PRs?

File: languages/lithuanian/apk/build.gradle [hunk 1]
```diff
@@ -0,0 +1,9 @@
+ext.shouldBePublished = true
```
Comment (menny) [inline languages/lithuanian/apk/build.gradle]: AnySoftKeyboard's play-store account does not own this pack, and cannot publish it.
Are you going to publish it?

**Comment by menny** [comment]
@rimas-kudelis  it looks good, and I'll gladly merge this. Just a question about the pack ownership (see inline)

**Comment by rimas-kudelis** [comment]
@menny I can transfer ownership of the pack to you or to @friesenkiwi. I'd love to keep access to publish it myself too if possible, but if not, that's fine by me.

Also, I commented about the signing key in my initial comment. Do you have it already or should I send it your way?

**Comment by rimas-kudelis** [comment]
@menny ping

**Comment by menny** [comment]
So, if you want me to publish, please email me the signing key to menny@evendanan.net

**Comment by menny** [comment]
I can't give access to publish ATM, but it will be automatically published once hitting _master_

**Comment by menny** [comment]
Also, until I have the keys and ser them up correctly in the secrets mechanism, I can not publish this. Can you switch the publish flag to `false`?

**Comment by rimas-kudelis** [comment]
@menny I've sent the key your way. Assuming that the actual transfer of ownership of the package on Google Play is the only thing that's left now, I guess we could wait with merging until that is done and avoid flipping the flag? Would that work for you?

**Comment by menny** [comment]
sure. Follow the guide here: https://support.google.com/googleplay/android-developer/answer/6230247?hl=en
I'll send you my details over email

**Comment by rimas-kudelis** [comment]
@menny could you resend me your details (play console email address and order id)? I was unable to transfer the ownership of the package using the details you provided me with previously. :(

File: languages/lithuanian/apk/build.gradle (hunk 1 above)
Comment (rimas-kudelis) [reply languages/lithuanian/apk/build.gradle]: I want to transfer the ownership of this pack to you.

**Comment by menny** [comment]
@rimas-kudelis I replied to your email about the ownership. Let me know if the new details work for you.

**Comment by rimas-kudelis** [comment]
Thanks! The app transfer is now pending.

**Comment by menny** [comment]
got it: https://play.google.com/store/apps/details?id=org.herrlado.ask.languagepack.lithuanian

**Comment by rimas-kudelis** [comment]
Now all that's left is to delete the Lithuanian branch. :)

**Pull request merged**